reset-account-nonce-script/README.md

```diff
@@ -0,0 +1,23 @@
+# Account deletion hardhat script
+
+This project demonstrates how to delete an already existing ECDSA account.
```

Comment:
Note that the perspective is from an EVM dev and not a Hedera dev.
Also provide context to say one application of this might be the reseting of an accounts nonce to allow flows that require initial nonces such as deterministic contract address cases

Reply:
Fixed.